src/shared/components/layouts/SecondaryItems.tsx

```diff
@@ -52,23 +36,16 @@ const getArrowX = (element: HTMLSpanElement) => {
 
 type Props = {
   display: boolean;
-  setDisplay: Dispatch<SetStateAction<boolean>>;
+  close: () => void;
 };
 
-const ToolsDashboard = ({ display, setDisplay }: Props) => {
-  const [count, failure] = useSelector<
-    RootState,
-    [count: number, failure: boolean]
-  >((state) => {
-    const unseenJobs = Object.values(state.tools).filter(
-      (job) => 'seen' in job && job.seen === false
-    );
-    return [
-      unseenJobs.length,
-      // at the moment, will always be false
-      unseenJobs.some((job) => job.status === Status.FAILURE),
-    ];
-  });
+const ToolsDashboard = ({ display, close }: Props) => {
+  const count = useSelector<RootState, number>(
+    (state) =>
+      Object.values(state.tools).filter(
+        (job) => 'seen' in job && job.seen === false
```

Comment:
```suggestion
     (job) =>
          'seen' in job &&
          job.seen === false &&
          [Status.FINISHED, Status.FAILURE, Status.ERRORED].includes(job.status)
```
Noticed that the green dot appears when I have in my jobs:
1. Successful and seen jobs
2. Running (unfinished) jobs

Reply:
Ah, good catch, that was left over from the previous approach and I forgot to adjust this bit of code. I have now followed your suggestions (but extracting the statuses to check outside of this function) so that should be fixed now.

Reply:
For some reason when mapping the same ID twice it won't remove the seen status when the duplicate job is viewed:
https://user-images.githubusercontent.com/4357962/125056859-912d8200-e0a0-11eb-9edd-834968d25a00.mov

Reply:
Looks like the it doesn't get marked as seen I'm guessing because the URL for both jobs is the same.

Reply:
Addressed.
Now links to a job set a location state containing the internalID that this link refers to in order to only mark this specific one as read. In the really rare case where this state would not be set (if coming directly to the result URL but still owning the job, so with it being in the user's storage), it would mark all the jobs whose remoteID is the one in the URL as read.

Reply:
Having @xwatkins review this as Dan is on holiday now. Dan reviewed the behaviour with me in a previous call, feel free to review only the implementation in this specific commit https://github.com/ebi-uniprot/uniprot-website/pull/410/commits/58bb93cdbf83ac66f335eff43f8fd3b3159a6075 as the rest as already been reviewed (or feel free to review more if you want! 😅 )

Reply:
Resolving this comment and merging this PR in as it holds back on further development on the Basket because of refactoring being done on the different contents of the sliding panel